main/transaction/index.ts

```diff
@@ -129,14 +131,16 @@ function populate(rawTx: TransactionData, chainConfig: Common, gas: GasData): Tr
     txData.gasFeesSource = GasFeesSource.Frame
   }
 
-  const maxPriorityFee = useFrameMaxPriorityFeePerGas
-    ? gas.price.fees.maxPriorityFeePerGas
-    : (rawTx.maxPriorityFeePerGas as string)
+  const maxPriorityFee =
+    useFrameMaxPriorityFeePerGas && gas.price.fees.maxPriorityFeePerGas
```

Comment:
just to clarify, these updates are just checking gas.price.fees.maxPriorityFeePerGas exists before using it -- do we know what problem this was solving?

Reply:
iirc there were cases where this wasnt defined for some reason like faulty block updates or something and the renderer was crashing